src/core_plugins/kibana/public/dashboard/dashboard_app.js

```diff
@@ -160,9 +160,6 @@ app.directive('dashboardApp', function ($injector) {
 
       timefilter.enableAutoRefreshSelector();
       timefilter.enableTimeRangeSelector();
-      dash.searchSource.highlightAll(true);
-      dash.searchSource.version(true);
-      courier.setRootSearchSource(dash.searchSource);
```

Comment:
With removing this, does the `searchSource` attached to `dash` still have any functionality at all? I think it's saved with the saved object, could we actually remove saving a `searchSource` altogether with a dashboard?

/cc @Stacey-Gammon

Reply:
Don't think so... would be great to see it go. Can do in another PR though, imo.  Or I can handle it too.